src/commands/AbstractStateModificationCommand.ts

```diff
@@ -141,8 +142,8 @@ export default abstract class AbstractStateModificationCommand extends Command {
         return enabledServiceNames.loadServicesNames(state);
     }
 
-    private handleHookResponse (result): string[] {
-        return result.successes.flatMap(success => success.result);
+    private handleHookResponse (result): string | undefined {
```

Comment:
is there a type for `result` or is it ok being implicit any?

Reply:
There should be. I have checked the documentation I couldn't find the type definitions for hook responses. I had to console log result to see the object properties.

Reply:
I meant for this method

given:
```ts
private handleHookResponse (result): string | undefined
```

the `result` parameter has implicit `any` type - I was wondering whether you can give this parameter a type?